integration_test/third_party_apps_data/applications/nvml/enable

```diff
@@ -0,0 +1,13 @@
+# The built-in Ops Agent already includes the nvml receiver in the default pipeline.
```

Comment:
Unfortunately the process metrics will not exist if no load is generated.

Reply:
That's right - sorry I forgot about that! 
Let's still keep only the comment at the top (as we may potentially use this `enable` file in the public doc `example configuration` section), and move the steps that generates the load to `integration_test/third_party_apps_data/applications/nvml/exercise`. 
Also, instead of use wget to download the `test_cuda_kernel.cc` file, we may want to hard code the source code in the `exercise` file, something like this: 
```
tee /etc/google-cloud-ops-agent/config.yaml > /dev/null << EOF
...the content of test_cuda_kernel.cc...
EOF
g++ -I/usr/local/cuda/include test_cuda_kernel.cc -L/usr/local/cuda/lib64 -lcublas -lcudart -o loopcudakernel
nohup ./loopcudakernel </dev/null >loopcuda.out 2>&1 &
echo "Submitted ./loopcudakernel as pid $!"
```
in the case when we need to modify this, we can run the integration test locally to verify the new change. WDYT?

Reply:
Oh, perfect.